MC/config/PWGHF/ini/tests/GeneratorHF_D2H_ccbar_and_bbbar_gap2_SCCR_pO.C

```diff
@@ -0,0 +1,189 @@
+int External(std::string path = "o2sim_Kine.root") {
+
+    int checkPdgQuarkOne{4};
+    int checkPdgQuarkTwo{5};
+    float ratioTrigger = 1./2; // one event triggered out of 2
+
+    std::vector<int> checkPdgHadron{411, 421, 431, 4122, 4132, 4232, 4332};
+    std::map<int, std::vector<std::vector<int>>> checkHadronDecays{ // sorted pdg of daughters
```

Comment:
for each decay channel, the pdg codes have to be sorted (as specified by the comment ;-)), unless you modify later in the test the comparison in lines `139-144`

Reply:
can you clarify the problem a bit more? We do here the same thing as in the OO macro. It is ensured that the PDG codes are sorted for each decay in l139 following:
```cpp
               for (auto &decay : checkHadronDecays[std::abs(pdg)]) {
                    std::sort(decay.begin(), decay.end());
                    if (pdgsDecay == decay || pdgsDecayAntiPart == decay) {
                        nSignalGoodDecay++;
                        break;
                    }
                }
 ```